oracles/parallelism/src/main/java/de/learnlib/oracle/parallelism/StaticParallelOracleBuilder.java

```diff
@@ -96,7 +100,7 @@ public StaticParallelOracleBuilder(Supplier<? extends MembershipOracle<I, D>> or
     }
 
     @Nonnull
-    public StaticParallelOracle<I, D> create() {
+    private Collection<? extends MembershipOracle<I, D>> create() {
```

Comment:
Return types should not contain wildcards, as this makes the code hard to use in user-land (they have to keep handling the wildcards themselves)

Reply:
Do you mean the `?` in front of `? extends MembershipOracle<I, D>`? Maybe this goes together with the previous comment, I actually don't know the best way to create `MealyStaticParallelOracle`, and `DFAStaticParallelOracle`.